content/en/docs/troubleshooting/wifi/_index.md

```diff
@@ -33,4 +33,13 @@ rtw_led_ctrl=1
 rtw_vht_enable=1
 rtw_power_mgnt=0
 rtw_enusbss=0
-```
\ No newline at end of file
+```
+
+# Wifi openmower-xx shows up after wlan1 is being used
+
+change /etc/comitup.conf
+```
+enable_appliance_mode: false
+primary_wifi_device: wlan1

```

Comment:
you sure wlan1 is correct here? It means "use wlan1 for hotspot"

In my setup (external usb card for extra coverage) I'm setting `primary_wifi_device: wlan0`, so we get hotspot on builtin card (that is less prone to issues) and main connectivity on external

Reply:
yes, it is correct , when you use wlan1 then comitup will use wlan0, broadcast openmower-xx and you can just use it., after that you get some nginx error. 
with enable_appliance_mode you disable wlan0 for wifi broadcast. 

so this is only an issue when you have 2 wifi devices.

Reply:
is this now verified correct? can I merge?

Reply:
I, frankly, don't know. In my setup I have predictive interface names enabled, udev mapping internal interface to wlan1 and comitup.conf just contains `primary_wifi_device: wlan0`. Meanwhile interfaces work correctly:

```
wlx00c0caae7845  IEEE 802.11bgn  ESSID:"*******"  Nickname:"<WIFI@REALTEK>"
          Mode:Managed  Frequency:2.452 GHz  Access Point: 0A:55:31:********
          Bit Rate:150 Mb/s   Sensitivity:0/0
          Retry:off   RTS thr:off   Fragment thr:off
          Power Management:off
          Link Quality=55/100  Signal level=-62 dBm  Noise level=0 dBm
          Rx invalid nwid:0  Rx invalid crypt:0  Rx invalid frag:0
          Tx excessive retries:0  Invalid misc:0   Missed beacon:0

wlan1     IEEE 802.11  Mode:Master  Tx-Power=31 dBm
          Retry short limit:7   RTS thr:off   Fragment thr:off
          Power Management:on
```

Without additional settings power management is off on external and internal is switched off.

I would create some badge, like "People tried that with success" that would mean that it's not official verified solution but something people can experiment with.

Reply:
it is by design when you have an external wifi device https://github.com/davesteele/comitup/issues/21
wlan0 stays up with comitup ip and broadcast wifi. if you connect you will get some nginx error.

on the openmower OS comitup.conf it also have now a reference to these 2 params